src/games/strategy/triplea/delegate/MoveDelegate.java

```diff
@@ -623,63 +631,6 @@ private void removeAirThatCantLand() {
   /**
    * This method is static so it can be called from the client side.
    *
-   * @param route
-   *        referring route
-   * @param units
-   *        referring units
-   * @param transportsToLoad
-   *        units to be loaded
-   * @param isload
-   * @param player
-   *        PlayerID
-   * @return a map of unit -> transport (null if no mapping can be
-   *         done either because there is not sufficient transport capacity or because
-   *         a unit is not with its transport)
-   */
-  public static Map<Unit, Unit> mapTransports(final Route route, final Collection<Unit> units,
```

Comment:
Remove this method as call calls had isload=true so boiled down to `mapTransportsToLoad`